histogram.js

```diff
@@ -171,6 +187,11 @@ function core(opt) {
     const dim = getDimensions(chartEl);
     // console.log(dim);
 
+    if (dim.contentWidth <= 0 || dim.contentHeight <= 0) {
```

Comment:
This wasn't necessary before. Did something change?

Reply:
That's not there before. I found that the chart is rendered before the tabs. That's an easy fix to avoid the order of rendering.

Reply:
But it used to work fine without this code, so why wasn't it necessary before?

Reply:
We had the same rendering problem before. Then we fixed it by adding and removing the `hidden` attribute. For this new Tabs, Tab, TabPanel. It also adds and removes the 'hidden'. But the histogram will calculate the `dim.contentWidth == 0 || dim.contentHeight == 0`. So I added this check there.

Reply:
Why does it calculate 0s now but it didn't before?

Reply:
I adapted the Tabs code from the reference you sent. It was integrated well except for the order of rendering. The histogram was trying to render before the <iobio-tab-panel>. I thought it was because I didn't initialize the size of the <iobio-tab-panel>. But that didn't work. 

And what's your concern about the check code there?

Reply:
My concern is in adding things without understanding why they became necessary. That fix feels too much like a hack to me. If there's a good explanation that's fine, but I don't feel like we fully understand what the problem is here.

Reply:
To explain further. If I add a print statement like this:

```javascript
const dim = getDimensions(chartEl);

console.log(dim.contentWidth, dim.contentHeight); // <- add this
```

I see that it's printing lots of 0, 0. If I add the same print statement to the main branch, no 0s. I want to understand why we're seeing 0 sizes all of a sudden.

Reply:
My concern is that you might be fixing the symptoms of the problem, without getting to the root. Leaving a root problem unfixed can lead to other symptoms later.

Reply:
That's a good point. I'll go back to look at the Tabs component. 

I might be in the wrong direction. It might be not the problem of rendering order. If you have any thoughts about the rendering order, please let me know.

Reply:
It should be fixed now. It turns out that the hidden attribute makes the element not visible and it is also removed from the document flow, which causes the histogram rendering problem. 

Instead of setting up the HTML "hidden" attribute, I added the CSS "visibility: hidden" to panels.

Reply:
Excellent